src/components/05_pages/NodeForm/index.js

```diff
@@ -63,7 +63,7 @@ class NodeForm extends React.Component {
               // @todo How do we handle cardinality together with jsonapi
               // making a distinction between single value fields and multi value fields.
               return React.createElement(this.props.widgets[widget], {
-                key: { fieldName },
+                key: fieldName,
```

Comment:
Updated the key as I kept getting the following error:

Warning: Encountered two children with the same key, `[object Object]`. Keys should be unique so that components maintain their identity across updates. Non-unique keys may cause children to be duplicated and/or omitted — the behavior is unsupported and could change in a future version.

Reply:
+1